From b30b17851ac56de269aa5ed0e23ae859279ca91e Mon Sep 17 00:00:00 2001
From: Dmitry Dementyev <dementyev@google.com>
Date: Mon, 6 Mar 2023 22:00:01 +0000
Subject: [PATCH] Convert argument to Intent in car settings account removal.

Bug: 265798353
Test: manual
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:a7a4b16b8ff00ced029c046786d295646e09ec2d)
Merged-In: Ie7ca80da84eb9592beee955244294fad1d90104d
Change-Id: Ie7ca80da84eb9592beee955244294fad1d90104d
---
 .../settings/accounts/AccountDetailsPreferenceController.java   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/car/settings/accounts/AccountDetailsPreferenceController.java b/src/com/android/car/settings/accounts/AccountDetailsPreferenceController.java
index 22542cf68..e8b1b7d98 100644
--- a/src/com/android/car/settings/accounts/AccountDetailsPreferenceController.java
+++ b/src/com/android/car/settings/accounts/AccountDetailsPreferenceController.java
@@ -61,7 +61,7 @@ public class AccountDetailsPreferenceController extends AccountDetailsBasePrefer
                     Intent removeIntent = result.getParcelable(AccountManager.KEY_INTENT);
                     if (removeIntent != null) {
                         done = false;
-                        getFragmentController().startActivityForResult(removeIntent,
+                        getFragmentController().startActivityForResult(new Intent(removeIntent),
                                 REMOVE_ACCOUNT_REQUEST, this);
                     } else {
                         success = future.getResult().getBoolean(AccountManager.KEY_BOOLEAN_RESULT);
-- 
2.40.0.634.g4ca3ef3211-goog

